provider/pkg/gen/nodejs-templates/helm/v2/helm.ts

```diff
@@ -203,7 +204,28 @@ export class Chart extends yaml.CollectionComponentResource {
                 const namespaceArg = cfg.namespace
                     ? `--namespace ${shell.quote([cfg.namespace])}`
                     : "";
-                let cmd = `helm template ${chart} --name-template ${release} --values ${defaultValues} --values ${values} ${apiVersionsArgs} ${namespaceArg}`;
+
+                // Check the helm version - v2 or v3
+                let helmVerCmd = `helm version --short`;
+                const helmVer = execSync(
+                    helmVerCmd,
+                    {
+                        env: {...process.env},
+                        maxBuffer: 512 * 1024 * 1024, // 512 MB
```

Comment:
Shouldn't need to set the env or maxBuffer here. Defaults are fine.

Reply:
Removed!